qiskit_experiments/library/calibration/frequency_cal.py

```diff
@@ -0,0 +1,112 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Ramsey XY frequency calibration experiment."""
+
+from typing import List, Optional
+
+from qiskit import QuantumCircuit
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework import ExperimentData, fix_class_docs
+from qiskit_experiments.library.characterization.ramsey_xy import RamseyXY
+from qiskit_experiments.calibration_management.backend_calibrations import BackendCalibrations
+from qiskit_experiments.calibration_management.update_library import Frequency
+from qiskit_experiments.calibration_management.update_library import BaseUpdater
+from qiskit_experiments.calibration_management.base_calibration_experiment import (
+    BaseCalibrationExperiment,
+)
+
+
+@fix_class_docs
+class FrequencyCal(BaseCalibrationExperiment, RamseyXY):
+    """A qubit frequency calibration experiment based on the Ramsey XY experiment.
+
+    # section: see_also
+        qiskit_experiments.library.characterization.ramsey_xy.RamseyXY
+    """
+
+    def __init__(
+        self,
+        calibrations: BackendCalibrations,
+        qubit: int,
```

Comment:
Can you invert the order here? All the other *Cal experiments have qubit before calibrations.

Reply:
Done here: d3bf230